sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/UnsupportedOperationsSuite.scala

```diff
@@ -62,7 +62,7 @@ class UnsupportedOperationsSuite extends SparkFunSuite with SQLHelper {
 
   assertNotSupportedInBatchPlan(
     "select on streaming source",
-    streamRelation.select($"count(*)"),
+    streamRelation.select($"`count(*)`"),
```

Comment:
Just a question. Do we need this change after this PR?

Reply:
No. I'll restore here. Thanks.

Reply:
Yes, this is necessary because `$""` in `catalyst/dsl` invokes `UnresolvedAttribute.apply`.